python/nano/tutorial/training/pytorch-lightning/run_nano_pytorch_lightning_test.sh

```diff
@@ -0,0 +1,9 @@
+export ANALYTICS_ZOO_ROOT=${ANALYTICS_ZOO_ROOT}
+export NANO_HOME=${ANALYTICS_ZOO_ROOT}/python/nano/src
+export NANO_TUTORIAL_TEST_DIR=${ANALYTICS_ZOO_ROOT}python/nano/tutorial/training/pytorch-lightning/
+
+set -e
+
+python $NANO_TUTORIAL_TEST_DIR/lightning_channel_last.py
```

Comment:
directly running the script might take a long time. In the inference example, we set the "fast_dev_run" argument to "True" using sed.